src/controller/config.tsx

```diff
@@ -14,8 +14,9 @@ const SNOOZE_OPTIONS: { [key in SnoozeReason]: SnoozeOption } = {
     snoozeReason: "Test Data - should be deleted", // Snooze Reason
     shortText: "Test Data",
     duration: 365,
-    type: null,
-    subType: null
+    followUp: "ServiceNow ticket ID:",
```

Comment:
Why is this being changed? Should we remove the "Test Data" object all together? Can't imagine the end users actually want to see this.

Reply:
We need the "Test data" snooze reason. Since we went to prod our users have classified some cases as ELIS test data (yes there is test data in production) and need to file a ServiceNow to have the case removed.